docs/browser-engine/webview2.md

```diff
@@ -0,0 +1,72 @@
+
+
+<!-- ✍ Please write about the technology, when it was created, what type of technology is it,
+some of the key characteristics for developers, etc. -->
+
+Althought the traditional IE WebBrowser control dates back to (2008 -- #JS can we find the actual date), WebView2 is a newer technology that has modernized the embeddable web browser by adopting the microsoft edge chromium platform. WebView2 has recently GA'd on Win32 C/C++, .NET, and expects to have UWP support soon. 
+
+WebView2 has both an evergreen and fixed distribution mode. With the default Evergreen mode, WebView2 will get the latest features, security, and bug-fixes automatically! For developers who want more control over their app updates we also offer the fixed version which allows you to ship webview2 with your app.
+
+WebView2 is backed by Microsoft, and continues to engage with the developer community and take feature requests and bug fixes.
+
+## Release cadence, version support and update model
+
+<!-- ✍ How often is there a new version? Is it predictable? Does it use an evergreen model (i.e.:
+it gets updated automatically) or do developers need to take care of that? Does it depend on other
+projects to ship? -->
+
+There are 2 parts to WebView2: the SDK, which comes in the form of a NuGet package, and the WebView2 Runtime, which take and optimize the edge browser binaries. The SDK and Runtime get updated every 4 weeks, in alignment with Edge. WebView2 offers a prerelease SDK with up and coming experimental features and a stable release SDK. 
+
+## Governance
+
+<!-- ✍ Please add an introductory paragraph about the governance model of the project: part of a
+foundation, company driven, etc. -->
+
+WebView2 is backed by Microsoft, and is built on top of the Edge browser platform. 
+
+## Summary
+
+| Characteristic |        |
+| -------------- | :----: |
+| Website | [{{technologies.webview2.url}}]({{technologies.webview2.url}}) |
+| Platforms | Desktop: Win32, WPF, WinForms |
+| Type | [Browser engine] |
+| Software type | close source |
+| License | https://www.nuget.org/packages/Microsoft.Web.WebView2/0.9.430/License |
+| Copyright | Micrsoft |
+| Documentation | [{{technologies.webview2.documentation}}]({{technologies.webview2.documentation}}) |
+| Community | [{{technologies.webview2.community}}]({{technologies.webview2.community}}) |
+| Latest version | {{technologies.webview2.releases.0.version}} |
+| Release cadence | Major versions: 4 weeks <br/> Minor/patch: 4 weeks |
+| Release support | 4 weeks |
+| Update model | Both Evergreen (automatic updates) or fixed model|
+| Governance model | WebView2 team + developer engagement |
+
+**Platform support:**
+
+{{ table technologies.webview2.platforms.{} }}
+
+:::note
+UWP support coming soon!
+:::
+
+**Language support:**
+
+{{ table technologies.electron.languages.{} }}
+
+<!-- Ref links -->
+
+[Browser engine]: ./browser-engine.md
```

Comment:
Please add all the links through the document here so it is easier to update and read the markdown 🙏